feat(SJFC-23): describe Job multiple deletion, update screenshots (#603)
mia-platform/documentation: docs/development_suite/monitoring/resources/jobs.md

![list_of_jobs](../img/jobs_list.png)

## Deleting Jobs

:::info

This feature is only available to users with sufficient permissions. For more information, consult the [Console Levels and Permission Management](/development_suite/identity-and-access-management/console-levels-and-permission-management.md) section of the docs.

:::

:::warning

The deletion is permanent, therefore make sure to save elsewhere any important information before deleting a Job.
Please note that also the related Pod(s) are marked for deletion, along with their information such as logs.

:::

You can delete a single Job by clicking on the **Delete button** on the last column of the Job table.

> You can also delete multiple Jobs at the same time, in this case select the checkbox on the left of the Jobs you want to delete and click on the **Delete Jobs** button that appears at the bottom of the table.

Reviewer: ```suggestion
You can also delete multiple Jobs at the same time, by selecting the checkbox on the left of the Jobs you want to delete and clicking on the **Delete Jobs** button that appears at the bottom of the table.
```
